rmf_traffic_ros2/src/rmf_traffic_ros2/schedule/Negotiation.cpp

```diff
@@ -949,6 +985,45 @@ class Negotiation::Implementation
 
     return std::make_shared<Handle>(for_participant, negotiators);
   }
+
+  void set_retained_history_count(uint count)
+  {
+    retained_history_count = count;
+  }
+
+  TableViewPtr table_view(
+    uint64_t conflict_version,
+    const std::vector<ParticipantId>& sequence)
+  {
+    const auto negotiate_it = negotiations.find(conflict_version);
+    if (negotiate_it == negotiations.end())
+    {
+      RCLCPP_WARN(node.get_logger(), "Conflict version %llu does not exist."
+        "It may have been successful and wiped", conflict_version);
+      return TableViewPtr();
```

Comment:
Nitpick: You can do `return nullptr;` here. I find that to be more explicit that a null value is being returned.

Reply:
fixed 94509f9